ios/RCTTWVideoModule.m

```diff
@@ -123,6 +123,71 @@ - (void)addLocalView:(TVIVideoView *)view {
   [self updateLocalViewMirroring:view];
 }
 
+- (void)captureLocalFrame:()string {
+  if (self.camera) {
+    // Set up the AVCaptureSession
+    AVCaptureSession *captureSession = [[AVCaptureSession alloc] init];
+    captureSession.sessionPreset = AVCaptureSessionPresetPhoto;
+
+    // Set up the AVCaptureDevice
+    AVCaptureDevice *captureDevice;
+    AVCaptureDevicePosition position = self.camera.device.position;
+    if (position == AVCaptureDevicePositionBack) {
+      captureDevice = [TVICameraSource captureDeviceForPosition:AVCaptureDevicePositionBack];
+    } else {
+      captureDevice = [TVICameraSource captureDeviceForPosition:AVCaptureDevicePositionFront];
+    }
+    NSError *error;
+    AVCaptureDeviceInput *input = [AVCaptureDeviceInput deviceInputWithDevice:captureDevice error:&error];
+    if (input) {
+      [captureSession addInput:input];
+    } else {
+      NSLog(@"Error setting up capture device input: %@", error.localizedDescription);
+      return;
+    }
+
+    // Set up the AVCapturePhotoOutput
+    AVCapturePhotoOutput *photoOutput = [[AVCapturePhotoOutput alloc] init];
+    [photoOutput setHighResolutionCaptureEnabled:YES];
+    if ([captureSession canAddOutput:photoOutput]) {
+      [captureSession addOutput:photoOutput];
+    } else {
+      NSLog(@"Error adding capture photo output");
+      return;
+    }
+
+    // Start the capture session
+    [captureSession startRunning];
```

Comment:
Will this work even though the camera is already being used for video capture?

Reply:
@joshdhenry I saw you approved this PR. Were you able to get the example app running and test it out?